tests/test_utils.py

```diff
@@ -0,0 +1,157 @@
+"""Tests for the 'utils' module."""
+
+import json
+from unittest import TestCase, mock
+from utils import execute_gremlin_dsl, select_latest_version,\
+    get_response_data, check_license_conflict
+
+
+@mock.patch("utils.get_session_retry")
+def test_check_license_conflict(mocker):
+    """Test the function check_license_conflict."""
+    payload = {"a": "b"}
+    mocker.return_value = MockedSession("lic_true")
+    out = check_license_conflict(payload)
+    print(out)
+    assert out is "false"
+
+    mocker.return_value = MockedSession("lic_false")
+    out = check_license_conflict(payload)
+    print(out)
```

Comment:
leftover?

Reply:
The print statements are purposefully kept so that in case the test case fails, at least we would be able to analyze via the print output.

Reply:
ok is not the assert output enough :)